content/docs/studio/index.md

```diff
@@ -22,10 +22,7 @@ repositories.
 
 </cards>
 
-https://www.youtube.com/watch?v=It74fReRHAU
-
-_Note that we have renamed DVC Studio mentioned in the above video to Iterative
-Studio and Views to Projects._
+https://www.youtube.com/watch?v=w-UjAbdpRY4
 
 **Why Iterative Studio?**
```

Comment:
```suggestion
**Why DVC Studio?**
```

Reply:
Are we doing the name changes to the docs under a separate PR?

Reply:
@jendefig I didn't change this because changing the name needs to done in other places too (eg, within Studio), and I was thinking that we would do it all together. Also, after yesterday's 1-1 with Dmitry, I got a sense that `DVC Studio` is not settled as the name yet. So I'll merge this PR now. If the name change is confirmed, I'll create another PR with the name changes to the docs.